demisto_sdk/commands/common/tests/tools_test.py

```diff
@@ -2435,13 +2473,15 @@ def test_xsoar_marketplace_version(self):
         assert "### Inline:" in actual
         assert "XSOAR" in actual
         assert "xsoar" in actual
-        assert self.MARKETPLACE_TAG_PARSER.XSOAR_PREFIX not in actual
-        assert self.MARKETPLACE_TAG_PARSER.XSIAM_PREFIX not in actual
-        assert self.MARKETPLACE_TAG_PARSER.XPANSE_PREFIX not in actual
+        assert self.XSOAR_PREFIX not in actual
+        assert self.XSIAM_PREFIX not in actual
+        assert self.XPANSE_PREFIX not in actual
+        assert self.XSOAR_SAAS_PREFIX not in actual
         assert "XSIAM" not in actual
         assert "xsiam" not in actual
         assert "XPANSE" not in actual
         assert "xpanse" not in actual
+        assert "XSOAR_SAAS" and "xsoar saas" not in actual
```

Comment:
Syntax mistake, the first condition (`"XSOAR_SAAS"`) always evaluates to `True`. 
Perhaps you meant `assert "XSOAR_SAAS" not in actual and "xsoar saas" not in actual`, which I'd replace with `assert "xsoar_saas" not in actual.lower()`

Reply:
Thanks for the catch fixed.